neverpile-fusion-jpa/src/main/java/com/neverpile/fusion/jpa/collection/index/MetadataIndexEntity.java

```diff
@@ -10,8 +13,10 @@
 
 @Entity
 @Table(name = "collection_metadata_index",
-    indexes = {@Index(name = "metadata_index",
-        columnList = "metadataPath, metadataValue")
+    indexes = {@Index(name = "metadata_index1",
+        columnList = "metadataPath, metadataValueString"), @Index(name = "metadata_index2",
+        columnList = "metadataPath, metadataValueTime"), @Index(name = "metadata_index3",
```

Comment:
Sprechende Namen auch für die Indexe?
```suggestion
        columnList = "metadataPath, metadataValueString"), @Index(name = "metadata_string_index",
        columnList = "metadataPath, metadataValueTime"), @Index(name = "metadata_datetime_index",
```

Reply:
das ist in der Autoformatirung verrutscht - die columns gehören immer zu dem Name in der Zeile drüber...